libs/ledger-live-common/package.json

```diff
@@ -55,12 +55,9 @@
     "https": false
   },
   "dependencies": {
-    "@celo/base": "^1.5.2",
-    "@celo/connect": "^1.5.2",
-    "@celo/contractkit": "^1.5.2",
-    "@celo/utils": "^1.5.2",
-    "@celo/wallet-base": "^1.5.2",
-    "@celo/wallet-ledger": "^1.5.2",
```

Comment:
☝️ It looks like `@celo/utils` and `@celo/connect` are required dependencies.  I got an error building LLC about these dependencies being missing when I tried to build this lib.

Reply:
I'll check it, thanks!